test/integration/Main.hs

```diff
@@ -269,6 +269,8 @@ testScenarioSolution _ci _em =
         , testSolution Default "Testing/1157-drill-return-value"
         , testSolution Default "Testing/1171-sniff-command"
         , testSolution Default "Testing/1171-chirp-command"
+        , testSolution Default "Testing/1171-resonate-command"
+        , testSolution Default "Testing/1205-goal-condition-exception"
```

Comment:
Not sure how we should set up a scenario-based test that, when the behavior is working correctly, the test should fail.

I think we would need to teach the test suite to differentiate between a scenario fail and a scenario win, and then add an extra "failure" goal condition that is "achieved" upon an exception.  When this behavior is fixed, then the normal win condition will be achieved instead.